pkg/resource/v1/chartconfig/resource.go

```diff
@@ -0,0 +1,66 @@
+package chartconfig
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/cluster-operator/pkg/cluster"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// Name is the identifier of the resource.
+	Name = "chartconfigv1"
+)
+
+// Config represents the configuration used to create a new chart config resource.
+type Config struct {
+	BaseClusterConfig *cluster.Config
```

Comment:
What is this and why do we need it?

Reply:
I wasn't sure on that. The resource will need to know the API server domain of the guest cluster. I thought I could get it from there but I'll remove and add it later if I need it.